src/pe_reports/data/db_query.py

```diff
@@ -52,7 +50,7 @@ def get_orgs(conn):
     """Query organizations table."""
     try:
         cur = conn.cursor()
-        sql = """SELECT * FROM organizations"""
+        sql = """SELECT * FROM organizations WHERE report_on"""
```

Comment:
Do any other scans or functions rely on this returning all orgs?

Reply:
No. Currently all the pe_source scripts and report generator should only run on orgs we're reporting on.